proto/prysm/v1alpha1/attestation.go

```diff
@@ -156,11 +192,29 @@ func (a *PendingAttestation) GetSignature() []byte {
 	return nil
 }
 
+// SetSignature --
+func (a *PendingAttestation) SetSignature(_ []byte) {}
+
 // Version --
 func (a *AttestationElectra) Version() int {
 	return version.Electra
 }
 
+// IsNil --
+func (a *AttestationElectra) IsNil() bool {
+	return a == nil ||
+		a.Data == nil ||
+		a.Data.Source == nil ||
+		a.Data.Target == nil ||
+		a.AggregationBits == nil ||
+		a.CommitteeBits == nil
+}
+
+// IsAggregated --
+func (a *AttestationElectra) IsAggregated() bool {
+	return a.AggregationBits.Count() > 1
+}
```

Comment:
I created these two methods because I ran into import cycles when trying to use existing helpers. That aside, I think having such methods is nicer.